cdk/src/open-data-platform/frontend/url-prefixes.ts

```diff
@@ -19,12 +19,12 @@ export default prefixes;
  */
 export function handler(event: { request: { uri: string } }) {
   // CloudFront Functions doesn't support const: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/functions-javascript-runtime-features.html
-  var request = event.request;
-  var beforeUrl = request.uri;
+  const request = event.request;
```

Comment:
I needed to do this or my page would not compile

Reply:
I'm pretty sure this will break the function. Did you successfully deploy and test that your sandbox can view tiles? If it does, then nvm and please remove the comment above this line.

If it does not, then what is the error the compiler gives you?

Reply:
The file is now out of the PR. Sorry about that